### PR TITLE
Adding support for singularity-ce in info command

### PR DESCRIPTION
**Pull request by rloewe**

### CODE REVIEW

File: cotainr/tests/cli/test_info.py [hunk 1]
```diff
@@ -127,6 +130,17 @@ def test_found_singularity(self, monkeypatch):
             == "Found singularity 3.7.4-1 >= 3.7.4, \x1b[92mOK\x1b[0m"
         )
 
+    def test_found_singularity_ce(self, monkeypatch):
```
Comment (rloewe) [inline cotainr/tests/cli/test_info.py:133]: I am not sure whether these tests are well-designed. We could reduce them to one with parameterization.

**Review by Chroxvi — CHANGES_REQUESTED** [review]
I have suggested a few minor changes.

File: cotainr/__init__.py [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "python": (3, 8, 0),
     "apptainer": (1, 0, 0),
     "singularity": (3, 7, 4),
+    "singularity-ce": (3, 8, 0),
```
[the remark this answers is not in the document]
Comment (rloewe) [reply cotainr/__init__.py:19]: SingularityCE is the version released by sylabs. They changed the name from singularity with version 3.8.0. That is why, I put that version in. I completely agree that we should test more versions of singularity.

[the remark this answers is not in the document]
Comment (Chroxvi) [reply cotainr/__init__.py:19]: Looking at the [Sylabs GitHub Singularity release page](https://github.com/sylabs/singularity/releases), it seems that the first release of "SingularityCE" was 3.8.0, as you have added here. I missed that our 3.8.7 release of "Singularity" is from the Apptainer repo.

However, Sylabs only started to provide deb-packages via GitHub from SingularityCE 3.9.1 and on. Thus, I think we should set 3.9.1 as our minimum requirement and add that as the version we test in CI - as a pragmatic solution for now.

Then at some point we should also add more recent versions of Singularity/Apptainer to our test matrix.

File: cotainr/tests/cli/test_info.py (hunk 1 above)
Comment (Chroxvi) [reply cotainr/tests/cli/test_info.py:133]: I will add a task to the backlog to refactor these tests. I think they are fine for now.

**Review by Chroxvi — APPROVED** [review]
I think this is ready to be merged.

**Pull request merged**